.github/workflows/update-website.yml

```diff
@@ -0,0 +1,21 @@
+name: update-website
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  trigger-website-build:
+    name: Trigger website build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Dispatch update-website event
```

Comment:
Can we reuse actions like this?

* https://github.com/marketplace/actions/workflow-dispatch
* https://github.com/marketplace/actions/dispatch-action

Reply:
Technically we could, but I feel that the Curl interface is more well known.

Note that GitHub has a native way to use workflow-dispatch via "uses:", so [the workflow-dispatch action](https://github.com/marketplace/actions/workflow-dispatch) may not add much value here, if at all.

Reply:
What's the reason we're not using the "uses:" function? The curl syntax is rather obscure...

Reply:
Discussed in person, let's go!